upgrade-instructions.md

```diff
@@ -29,9 +29,13 @@ Date 26/11/2019
 
 SHA-1 0bf4e8384e
 
+Concerned products: Guice distributed James (rabbitMQ)
+
 The distributed James project (relying on Guice, Cassandra, ElasticSearch, RabbitMQ and optionally Swift) benefits from a new distributed task mananger.
 
-For the plain operation if this new task manager, James now relies at least on RabbitMQ 3.8. So please upgrade your RabbitMQ version if you use this component.
+In order to enforce task sequential processing, we rely on an exclusive consumer, which is a feature introduced in RabbitMQ 3.8.
```

Comment:
s/an exclusive consumer/a single active consumer

Reply:
Naming the rabbitMQ 3.8 feature explicitly is intendeed @remk .

Maybe we can say : `In order to enforce task sequential processing via a signle active consumer at the cluster level, we rely on an exclusive consumer, which is a feature introduced in RabbitMQ 3.8`